docs/source/FAQ.md

```diff
@@ -662,49 +649,41 @@ data:
     dummy:
         path_src: data/train/data.src
         path_tgt: data/train/data.tgt
-        src_feats:
-            feat_0: data/train/data.src.feat_0
-            feat_1: data/train/data.src.feat_1
-        transforms: [filterfeats, onmt_tokenize, inferfeats, filtertoolong]
+        transforms: [onmt_tokenize, inferfeats, filtertoolong]
         weight: 1
     valid:
         path_src: data/valid/data.src
         path_tgt: data/valid/data.tgt
-        src_feats:
-            feat_0: data/valid/data.src.feat_0
-            feat_1: data/valid/data.src.feat_1
         transforms: [filterfeats, onmt_tokenize, inferfeats]
```

Comment:
`filterfeats` was removed, wasn't it?

Reply:
Yes. Source features are now appended to source tokens ` I￨1￨3 love￨0￨1 eating￨0￨1 pizza￨0￨1`